test/EVMHost.cpp

```diff
@@ -39,17 +39,18 @@ using namespace evmc::literals;
 
 evmc::VM& EVMHost::getVM(string const& _path)
 {
-	static evmc::VM theVM;
-	if (!theVM && !_path.empty())
+	static evmc::VM NullVM{nullptr};
+	static std::map<string, evmc::VM> vms;
```

Comment:
Better use `map<string, unique_ptr<evmc::VM>>`, otherwise the returned reference might be invalidate if this function is called a second time.